themes/notmyidea/templates/base.html

```diff
@@ -71,10 +71,10 @@
 
         <footer id="contentinfo" class="body">
                 <address id="about" class="vcard body">
-                Proudly powered by <a href="http://getpelican.com/">Pelican</a>, which takes great advantage of <a href="http://python.org">Python</a>.
+                    Women in Computer Science (WiCS) is the undergraduate sub-committee of the University of Waterloo Women in Computer Science Committee. Check out their <a href="https://cs.uwaterloo.ca/wics" target="_blank">website</a>!
```

Comment:
I would make this "Women in Computer Science Undergraduate (WiCS ugrad)" maybe?

Also, don't think `target="_blank"`is necessary on the link; at minimum, it's inconsistent with the rest of the site.